addons/account/models/chart_template.py

```diff
@@ -1089,13 +1089,42 @@ def _get_account_reconcile_model(self, template_code):
     # Tooling
     # --------------------------------------------------------------------------------
 
-    def ref(self, xmlid, raise_if_not_found=True):
-        if '.' in xmlid:
-            return self.env.ref(xmlid, raise_if_not_found)
-        return (
-            self.env.ref(f"account.{self.env.company.id}_{xmlid}", raise_if_not_found=False)
-            or self.env.ref(f"account.{self.env.company.parent_ids[0].id}_{xmlid}", raise_if_not_found)
-        )
+    def ref(self, xml_id, raise_if_not_found=True):
+        return self.refs([xml_id], raise_if_not_found)
+
+    def refs(self, xml_ids, raise_if_not_found=True):
```

Comment:
Hello, just a suggestion to ease the usage, could we maybe have have ids as *args to avoid adding extra (unnecessary) arrays everywhere in code?

And force `raise_if_not_found` to be a keyword param?

Reply:
See my other comment in odoo.api